### DISCUSSION TITLE
Accessing Camera QR code and barcode reading capabilities

### DISCUSSION THREAD

**Discussion by pierre01** [CommunityToolkit/Maui#183] [New Feature Discussions]
For a while now, all mobile devices have integrated QR code reading functions with their Cameras.
Since Covid, all restaurants have Menus in the form of a  QR code.
Many people have business cards with QR code with contact and business information.
Can we add this functionality to the toolkit?

**Comment by VladislavAntonyuk** [comment]
It will require additional dependency on https://github.com/micjahn/ZXing.Net.
you can use https://github.com/Redth/ZXing.Net.Mobile for Xamarin.Forms and https://github.com/Redth/BigIslandBarcoding for MAUI

**Comment by CliffAgius** [comment]
Closed as answered and also the CameraView port is a work in progress Issue #259
